### PR TITLE
Prettier setup

### PR DESCRIPTION
**Pull request by Rin-Akaia-eth**

### CODE REVIEW

File: src/components/export/export.js [hunk 1]
```diff
@@ -1,46 +1,49 @@
...
-import { InputAdornment } from '@mui/material';
-import Icon from '@mui/material/Icon';
-import TextField from '@mui/material/TextField';
-import { Base64 } from 'js-base64';
-import React, { Component } from 'react';
-import { ArgsAccount, ArgsBig, ArgsError, ArgsString } from '../../utils/args';
-import { errorMsg } from '../../utils/errors';
-import { STORAGE } from '../../utils/persistent';
-import { convert, toGas, toNEAR } from '../../utils/converter';
+import EditOutlinedIcon from "@mui/icons-material/EditOutlined";
```
Comment (Rin-Akaia-eth) [inline src/components/export/export.js:1]: The file is affected by formatter while merging `main` to this branch

File: src/components/export/export.js [hunk 2]
```diff
@@ -1,46 +1,49 @@
...
+        notWhitelisted: new ArgsError(
+            "Token not whitelisted on multicall instance",
+            (value) => this.errors.notWhitelisted
+        ),
+        hasErrors: new ArgsError("Please fix all errors", (value) => this.errors.hasErrors),
     };
 
     total = {
-        gas: new ArgsBig("267.5", toGas("1"), toGas("270"), "Tgas"),
+        gas: new ArgsBig("270", toGas("1"), toGas("270"), "Tgas"),
```
Comment (mohamedalichelbi) [inline src/components/export/export.js:38]: Why did it revert the changes from 87368ecd9f6583af091af0001147f6f9590844a4 ?

Comment (Rin-Akaia-eth) [reply src/components/export/export.js:38]: Oops, seems like merging glitch. I'll fix this

Comment (Rin-Akaia-eth) [reply src/components/export/export.js]: Fixed

**Review by mohamedalichelbi — APPROVED** [review]
LGTM! I added a pre-commit hook to make sure staged files are well formatted.
